internal/endorser/endorser_test.go

```diff
@@ -70,12 +74,15 @@ func TestGenerateEndorsement_MultipleValidEndorsement(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Could not load provenance: %v", err)
 	}
-	tempPath2, err := copyToTemp("../../testdata/provenance.json")
+	tempPath2, err := copyToTemp("../../testdata/provenance_variant.json")
```

Comment:
@rbehjati I think in this test we just over-wrote the `provenance.json` in temp before. I created a variant of `provenance.json`. Does this make sense?

Reply:
The previous approach was more flexible. Now we have an additional test file to maintain, and it does not add much value.  Is there any difference between `provenance.json` and `provenance_variant.json`?